spec/behavior/fileLoader.spec.js

```diff
@@ -0,0 +1,22 @@
+const { proxyquire, sinon } = testHelpers;
+
+describe( "fileLoader", () => {
+	it( "should cache file text", async () => {
+		const expected = "FIRST";
+		const fs = {
+			readFile: sinon.stub()
+		};
+		fs.readFile.onCall( 0 ).callsArgWith( 2, null, expected );
+		fs.readFile.onCall( 1 ).throws( new Error( "you failed" ) );
```

Comment:
Rather than setting different behaviors for different calls, would it be simpler just to assert that `fs.readFile` is called only once?